src/Console/Database/FixHtmlEncodingCommand.php

```diff
@@ -258,25 +303,56 @@ private function dumpObjects()
 
     private function fixItems()
     {
-        $itemtype = $this->input->getOption('itemtype');
-        $item_ids = $this->input->getOption('id');
-
-        foreach ($item_ids as $item_id) {
-            $item = new $itemtype();
-            if (!$item->getFromDB($item_id)) {
-                $this->failed_items[] = $item_id;
-                continue;
+        foreach ($this->invalid_items as $itemtype => $items) {
+            foreach ($items as $item_id => $fields) {
+                $item = new $itemtype();
+                if (!$item->getFromDB($item_id)) {
+                    $this->failed_items[$itemtype][$item_id] = $item;
+                    continue;
+                }
+                $url = $this->getItemUrl($item);
+                $this->output->writeln(
+                    '<comment>' . sprintf(__('About to fix itemtype %s ID %s - %s'), $itemtype, $item_id, $url) . '</comment>',
+                    OutputInterface::VERBOSITY_QUIET
+                );
+                if ($this->confirm) {
+                    $this->askForItemFix(false);
+                }
```

Comment:
If user ask to not confirm each item, I think it is not necessary to print a line for every item in quiet mode.
I suggest to print debug message in verbose only when confirmation is not required, and to adapt confirmation question otherwise.
```suggestion
                if (!$this->confirm) {
                    $this->output->writeln(
                        '<comment>' . sprintf(__('Fixing %s with ID "%s" (%s)'), $itemtype::getTypeName(1), $item_id, $url) . '</comment>',
                        OutputInterface::VERBOSITY_VERBOSE
                    );
                } else {
                    $this->askForItemFix(false);
                }
```